util/fibers/detail/fiber_interface.h

```diff
@@ -64,6 +64,9 @@ class FiberInterface {
   FI_SleepHook sleep_hook;
 
   ::boost::context::fiber_context SwitchTo();
+  void PrintStackTrace();
+
+  static void PrintAllFiberStackTraces();
```

Comment:
Would we be able to call this upon crash, via `absl::InstallFailureSignalHandler()`? I can see cases in which this might turn useful, but perhaps this will cause another signal to be raised, if we're already crashing?

Reply:
Maybe; I'm not super sure how well we can count on everything to still be up :/